.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
```

Comment:
I've not used pre-commits very often. Is this pre-commit verifying that there is is not any swiftlint errors/warnings? Should this be pointing at our ML swiftlint config?